chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/certifiedProduct/CertifiedProductHandler2014Version1.java

```diff
@@ -64,7 +62,33 @@ public TemplateColumnIndexMap getColumnIndexMap() {
     }
     
     public String[] getCriteriaNames() {
-        return this.criteriaNames;
+    	criteriaList.put("a", a);
```

Comment:
I feel like this sacrificed some readability and I'm not sure what the benefit is instead of just writing out the criteria names the way they were. If it's important for some other change in the code that's fine and I wouldn't hold up the PR for it but it's definitely harder to understand!

Reply:
Agreed. In two months we'll have no idea what these are doing without having to re-read through way too much stuff